phalcon/validation/validator/uniqueness.zep

```diff
@@ -185,52 +186,97 @@ class Uniqueness extends CombinedFieldsValidator
 				}
 			}
 
-			let attribute = this->getColumnNameReal(record, this->getOption("attribute", singleField)),
-				except = this->getOption("except");
+			let isModel = record instanceof Model;
+
+			if !(isModel || record instanceof Collection) {
```

Comment:
Not sure if parser issue. Well i mean if we could do something `!instanceof` could be nice.